app/controllers/admin/users_controller.rb

```diff
@@ -10,6 +10,19 @@ def index
       @users = User.all
     end
 
+    def toggle_confirmation
+      if params[:user][:should_be_confirmed] == 'true'
+        params[:user].delete(:should_be_confirmed)
+        params[:user][:confirmed_at] = Time.now
+        @user.update_attributes(user_params)
+      elsif params[:user][:should_be_confirmed] == 'false'
+        params[:user].delete(:should_be_confirmed)
```

Comment:
It's a bit messy :stuck_out_tongue: . @differentreality Is it possible to send different values other than 'true' and 'false' from the Bootstrap-Switch-Checkbox? I guess it would really help shorten the code.